meta-fobnail-distro/recipes-fobnail/fobnail-attester/files/fobnail_usb.network

```diff
@@ -0,0 +1,6 @@
+[Match]
```

Comment:
@kotylamichal network files are installed in systemd-conf recipe so I think it should go there

Reply:
Ok, but even if it is strictly related to fobnail attester configuration? moved to `systemd-conf_%.bbappend`